src/runtime.js

```diff
@@ -456,6 +456,19 @@ export function adoptShadowMode(global) {
  * @return {boolean}
  */
 function maybeLoadCorrectVersion(win, fnOrStruct) {
+  if (IS_ESM) {
```

Comment:
Actually, please move this to below the `typeof fnOrStruct == 'function'` check.

Reply:
done.